### PR TITLE
feat: Redesign FormCard component

### PR DESCRIPTION
**Pull request by flacial**
Related to #1734

This PR redesigns the FormCard component by following the [design mock](https://www.figma.com/file/f9xRyl1bJOjtBRd8zFORzg/c0d3?node-id=204%3A4568). 

It does the following:

- Create a new submit button style that could be enabled by passing the boolean `newBtn` prop
- Specify the alignment of elements by passing the `align` property that takes `right`, `left`, and `center`.
- Remove the card and `mdInput` backgrounds by passing the boolean `noBg` prop.
- Create custom styles for the FormCard component
- Add an icon to the error message
- Extract the error message to its own component

By merging this PR, it'll be possible to create the [AdminLessonInputs component](https://www.figma.com/file/f9xRyl1bJOjtBRd8zFORzg/c0d3?node-id=204%3A4568)

Before:

image

After:

image

### CODE REVIEW

File: components/FormCard.tsx [hunk 1]
```diff
@@ -127,18 +149,35 @@ export const FormCard: React.FC<FormCardProps> = ({
...
+        <div className={`text-center align-self-${textAlignment}`}>
+          {newBtn ? (
+            <button
+              onClick={btnOnClick}
+              className={styles.formCard__submitButton}
+            >
+              {onSubmit.title}
+            </button>
+          ) : (
+            <Button onClick={btnOnClick} type="primary" color="white">
```
Comment (flacial) [inline components/FormCard.tsx:177]: I kept the old button because I wasn't sure if it should be removed. My thought is to keep it as the default one.

**Review by SahilKalra98 — APPROVED** [review]

File: scss/formCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
+$light-gray: hsl(0, 0%, 85%);
+$mint-green: hsl(170, 66%, 38%);
```
Comment (songz) [inline scss/formCard.module.scss:2]: should `light-gray` and `mint-green` be more global, or are they exclusively going to be used in `formCard`?

**Review by songz — APPROVED** [review]

File: scss/formCard.module.scss (hunk 2 above)
Comment (flacial) [reply scss/formCard.module.scss:2]: I felt it was a different task to modify `_variables.scss`. But you're right, `$mint-green` should be global. `$light-gray` is exclusive for the `FormCard`.

File: scss/formCard.module.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+.optionInfo__error {
+  font-size: 14px;
+  display: flex;
+  align-items: center;
+  column-gap: 7px;
+  margin-left: 5px;
+}
+
+.formCard__submitButton {
```
Comment (flacial) [inline scss/formCard.module.scss:23]: Should be `formCard--submitButton` because it's a modifier.

**Pull request merged**